src/engine/loopingcontrol.cpp

```diff
@@ -742,7 +744,6 @@ void LoopingControl::slotBeatLoop(double beats, bool keepStartPoint) {
                     //
                     // If we press 1/2 beatloop we want loop from 50% to 100%,
                     // If I press 1/4 beatloop, we want loop from 50% to 75% etc
-                    double nextBeat = floor(m_pBeats->findNextBeat(cur_pos));
```

Comment:
Hm, it looks like this didn't check for nextBeat ~= prevBeat. Do we know if that was a bug or intended? (looks like a bug since it would produce a divide-by-zero below).